.gitignore

```diff
@@ -25,4 +25,4 @@ package-lock.json
 
 .vscode/
 *-workspace
-.npmrc
+**/*/.npmrc
```

Comment:
what's this for?

Reply:
I had to include the root `.npmrc` to get the package published initially. I still ignored all of the subdirectory .npmrcs just in case...

Like I said in the previous comment we could still ignore all of the .npmrc's when we keep the workspaces approach.